app/controllers/api/base_controller/authentication.rb

```diff
@@ -7,10 +7,10 @@ module Authentication
       def require_api_user_or_token
         log_request_initiated
         @auth_token = @auth_user = nil
-        if request.headers['X-MIQ-Token']
-          authenticate_with_system_token(request.headers['X-MIQ-Token'])
-        elsif request.headers['X-Auth-Token']
-          authenticate_with_user_token(request.headers['X-Auth-Token'])
+        if request.headers[HttpHeaders::MIQ_TOKEN]
```

Comment:
why change the strings used ?  "HTTP_X_MIQ_TOKEN" vs. "X-MIQ-Token", are they both defined in the headers ?